packages/protocol/test-sol/unit/governance/network/Governance.t.sol

```diff
@@ -3164,7 +3281,7 @@ contract GovernanceTest_hotfixWhitelistValidatorTally is GovernanceTest {
     assertEq(governance.hotfixWhitelistValidatorTally(HOTFIX_HASH), 3);
   }
 
-  function test_Reverts_WhenCalledOnL2() public {
+  function test_Reverts_WhenL2() public {
```

Comment:
Ditto. Might need to use a `_Setup` pattern, as in [ValidatorsTest_SetValidatorScoreParameters](https://github.com/celo-org/celo-monorepo/blob/release/core-contracts/12/packages/protocol/test-sol/unit/governance/validators/Validators.t.sol#L674).

Reply:
implemented in https://github.com/celo-org/celo-monorepo/pull/11267/commits/b2bff8fae4b704b2e479784e8621ca7ce5a96d72